### ISSUE TITLE
Mice arent sharded when running into the shard

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#19534]
(WEB REPORT BY: mask2697 REMOTE: 172.93.109.202:7777)
# Revision
4a8bd3e7ae934b0fab2c1454c643ae426a9f8b0d

# Description

Running into the supermatter as a mouse doesnt shard you
# Steps to Reproduce
be mouse, run into shard
# What you Expected
death
# What Actually Happened
cucked

**Comment by Exxion** [comment]
This is to prevent mouse-based power, since mice can respawn endlessly
Although it would be better if they still got vaporized but just didn't produce power

**Comment by jknpj** [comment]
After looking at the code seems like this is indeed a bug, they were supposed to JUST not make power but a goofy return makes them unshardable.

**Comment by Exxion** [comment]
Well nevermind then

**Comment by SGuivenne** [comment]
Replace the not a bug with Bug/Fix then?

**Comment by Exxion** [comment]
Right, I was on mobile when I posted that and forgot to change it when I got back to the computer
